lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -219,34 +219,48 @@ void StagesSubject::stageChanged(UsdNotice::ObjectsChanged const& notice, UsdSta
 			if (!sceneItem)
 				continue;
 
-			if (prim.IsActive())
+			// Special case when we know the operation came from either
+			// the add or delete of our UFE/USD implementation.
+			if (InAddOrDeleteOperation::inAddOrDeleteOperation())
```

Comment:
As we discussed, if we are in an add/delete prim operation, we'll send the appropriate add/delete UFE notif.